c/kastore.h

```diff
@@ -83,6 +83,10 @@ The requestion function cannot be called in the current mode.
 The requested type does not match the type of the stored values.
 */
 #define KAS_ERR_TYPE_MISMATCH                         -13
+/**
+End of file was reached without reading any data.
```

Comment:
This isn't quite right. How about "End of file was reached while reading data"?

Reply:
The error is only returned when `fread()` returns zero bytes. I can see your point though, because its possible we're part way into the kastore.